tests/zkasm/assert.js

```diff
@@ -0,0 +1,68 @@
+const fs = require("fs");
+
+class InstrumentInst {
+    /**
+     * @param {Object} config 
+     * @param {string} [config.outputFile] - the file to which output will be written
+     */
+    constructor(config) {
+        this.config = config;
+        this.results = {};
+    }
+
+    setup() {
+        // `zkevm-proverjs` requires a `setup` function on helper objects.
+    }
+
+    /**
+     * 
+     * @param {Object} ctx - context of zkasm program
+     * @param {Object} tag - information of helper function. 
+     * Helper function must be used in format:
+     * ${assert_eq(regname, regname, tag)}, for example:
+     * ${assert_eq(A, B, test_generated_from_line_3000)}
+     */
+    eval_assert_eq(ctx, tag) {
+        // TODO print verbose error on invalid params
+        // TODO: why arg* is list of two integers with last zero and first value in register???
+        // Is it chunks? If yes they should be merged instead of ignoring second one
```

Comment:
As I see it is really chunks and total number can be calculated as ``ctx[tag.params[idx].regName][0] + ctx[tag.params[idx].regName][1] * 2**32`` But maybe it worth to ask Polygon team?